cms/templates/widgets/header.html

```diff
@@ -98,6 +99,9 @@ <h3 class="title"><span class="label"><span class="label-prefix sr">${_("Course"
                   <li class="nav-item nav-course-settings-advanced">
                     <a href="${advanced_settings_url}">${_("Advanced Settings")}</a>
                   </li>
+                  <li class="nav-item nav-course-settings-certificates">
```

Comment:
Sorry, I just saw this. 
Sure i will add the check also. I will update you in my upcoming PR.